src/WebSocket.purs

```diff
@@ -15,22 +15,22 @@ module WebSocket
   , isBinary
   ) where
 
-import Prelude ((*>), Unit, class Applicative, (<<<), pure, unit, ($), class Semigroup, class Monoid, mempty, (>>=), class Bind)
-import Data.Nullable (Nullable, toMaybe, toNullable)
+import Data.Argonaut (class DecodeJson, class EncodeJson, Json, decodeJson, encodeJson, jsonParser, printJsonDecodeError, stringify)
+import Data.ArrayBuffer.Types (ArrayBuffer)
+import Data.Either (Either(..))
+import Data.Generic.Rep (class Generic)
 import Data.Maybe (Maybe)
-import Data.Either (Either (..))
-import Data.Argonaut (class EncodeJson, class DecodeJson, encodeJson, decodeJson, jsonParser, stringify, Json)
+import Data.Nullable (Nullable, toMaybe, toNullable)
 import Data.Profunctor (class Profunctor)
-import Data.Generic.Rep (class Generic)
-import Data.ArrayBuffer.Types (ArrayBuffer)
-import Data.Symbol (SProxy (..), reflectSymbol, class IsSymbol)
-import Web.File.Blob (Blob)
-import Foreign (Foreign)
+import Data.Symbol (SProxy(..), reflectSymbol, class IsSymbol)
 import Effect (Effect)
-import Effect.Uncurried (EffectFn1, EffectFn2, runEffectFn1, mkEffectFn1, mkEffectFn2)
-import Effect.Exception (Error, throw)
 import Effect.Class (class MonadEffect, liftEffect)
-import Type.Proxy (Proxy (..))
+import Effect.Exception (Error, throw)
+import Effect.Uncurried (EffectFn1, EffectFn2, runEffectFn1, mkEffectFn1, mkEffectFn2)
+import Foreign (Foreign)
+import Prelude ((*>), Unit, class Applicative, (<<<), pure, unit, ($), class Semigroup, class Monoid, mempty, (>>=), class Bind)
+import Type.Proxy (Proxy(..))
+import Web.File.Blob (Blob)
```

Comment:
My editor automatically sorts imports -- if you want them to be returned to the original formatting let me know and I can revert this.